lib/utils/convertToDailyOverview.js

```diff
@@ -0,0 +1,52 @@
+const { createTime } = require("./convertTimestamp");
+
+const buildDailyOverview = (documents, { now }) => {
+  // delay enabled to filter events up to 6 hour
+  const delay = 21600;
+
+  const dailyOverview = [
+    { title: "Next Up", list: true, content: [] },
+    { title: "Events today", list: false, content: [] },
+  ];
```

Comment:
```suggestion
  const nextUp = { title: "Next Up", list: true, content: [] };
  const eventsToday = { title: "Events today", list: false, content: [] }
  const dailyOverview = [
    nextUp ,
    eventsToday,
  ];
```

This makes this line easier to read this line:

`dailyOverview[0].content.push(info);` 
-> 
`nextUp.content.push(info);`

Reply:
absolutely, thanks